contrib/openssl-cmake/CMakeLists.txt

```diff
@@ -1457,6 +1457,11 @@ else()
     add_library(ssl ${SSL_SRC})
 endif()
 
+if (ENABLE_OPENSSL_FIPS)
+    target_compile_definitions(crypto PRIVATE -DOPENSSL_FIPS)
```

Comment:
I grepped `OPENSSL_FIPS` in contrib/openssl/ but could not find meaningful matches.

Are you sure we need to compile the crypto and ssl libraries with the define on?

Reply:
Huh, I don't know where I took it from. Let's remove.